docs/_data/sidebar.yml

```diff
@@ -20,22 +20,20 @@
 - title: User Guide
```

Comment:
Lets move the article on Prettier into this section. It currently is in Getting Started which I dont think is right.

Reply:
Moved Prettier code formatter to user guide section.